src/main/java/com/database/operations/CompanyCrud.java

```diff
@@ -0,0 +1,85 @@
+package com.database.operations;
+
+import com.database.data.Company;
+import com.database.data.DBCloseConnect;
+import com.database.data.JdbcConnectionUtil;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyCrud {
+
+    // Qwery1 - SELECT_ALL, Qwery2 - SELECT_ID , Qwery3 - INSERT;  Qwery4 - DELETE;  Qwery5 - UPDATE;
```

Comment:
Think about better names for constants and remove this comment.
Don't forget about java style and naming. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names